_posts/en/newsletters/2024-02-14-newsletter.md

```diff
@@ -0,0 +1,200 @@
+  ideas for allowing individual transactions to opt-in certain mempool
+  policies after [cluster mempool][topic cluster mempool] support has
+  been fully implemented, tested, and deployed.  The improvements build
+  on the features of [v3 transaction relay][topic v3 transaction relay]
+  by relaxing some of its rules that may no longer be needed and adding
+  a requirement that a transaction (or a [package][topic package relay]
+  of transactions) pay a feerate that makes them likely to be mined
+  within the next block or two.
+
+- **What would have happened if v3 semantics had been applied to anchor outputs a year ago?**
+  Suhas Daftuar [posted][daftuar retrospective] to Delving Bitcoin about
+  his research into the idea of automatically applying [v3 transaction
+  relay policy][topic v3 transaction relay] to [anchors-style][topic
+  anchor outputs] LN commitment and fee-bumping transactions (see
+  [Newsletter #286][news286 imbued] for the underlying _imbued v3_
+  proposal).  In short, he recorded 14,124 transactions in 2023 that
+  looked like anchor spends.  Of those,
+
+  - About 94% <!-- (14124 - 856) / 14124 --> would have been successful
+    under the v3 rules.
+
+  - About 2.1% <!-- 302/14124 --> had more than one parent (e.g.,
+    attempts to batch CPFP spends).  LN wallets do this for efficiency
+    when closing more than channel within a short amount of time.  They
+    would need to disable this behavior if anchor-style outputs were to
+    be imbued with v3 properties.
+
+  - About 1.8% <!-- 251/14124 --> were not the first child of the
+    parent.  Using the proposal for imbued v3, the second child would be
+    able to replace the first child in a [package][topic package relay].
+
+  - About 1.2% <!-- 175/14124 --> were apparently grandchildren of the
+    commitment transaction, i.e. spends of the spend of the anchor
+    output.  LN wallets might do this for a variety of reasons, from
+    closing multiple anchor channels in sequence to opening new channels
+    with their change from the anchor close.  LN wallets would be unable
+    to use this behavior if anchor-style outputs were imbued with v3
+    properties.
+
+  - About 1.2% <!-- 173 / 14124 --> were never mined and weren't
+    analyzed further.
+
+  - About 0.1% <!-- 19/14124 --> spent an unrelated unconfirmed output,
+    resulting in the anchor spending having more than the allowed one
+    parent.  Developer Bastien Teinturier thinks that this may have been
+    a behavior of Eclair and notes that Eclair would resolve this
+    situation automatically even with its current code.
+
+  - Less than 0.1% <!-- 10/14124 --> were larger than 1,000 vbytes.
+    This is also behavior that LN wallets would need to change.
+    Daftuar's further research showed that nearly all anchor spends were
+    less than 500 vbytes, potentially suggesting that the v3 size limit
+    could be reduced.  This would make it less expensive for a defender
```

Comment:
Relevant?
https://bitcoinops.org/en/newsletters/2024/01/03/#v3-transaction-pinning-costs

Reply:
Yes, but IIRC a couple reviewers on that newsletter found my description a bit hard to follow, so I think I'm going to stick with just the link to the pinning topic page.  Thanks for suggesting and finding that link, though!